### PR TITLE
Better prints for going to operation enabled

### PR DESCRIPTION
**Pull request by martijnvandermarel**
reduced print rate from every 0.1 sec to every 0.5 sec
add prints before and after indicating which joint is being turned on
add indentation to make it more clear to which joint all these prints belong

these prints have bothered me for a while, imo this is much better
ofc this is bit personal preference, if you have ideas about it feel free to tell me/change stuff

### CODE REVIEW

File: march_hardware/src/Joint.cpp [hunk 1]
```diff
@@ -55,7 +55,9 @@ void Joint::prepareActuation()
 {
   if (this->allowActuation)
   {
+    ROS_INFO("Trying to prepare joint %s for actuation", this->name.c_str());
```
Comment (Ishadijcks) [inline march_hardware/src/Joint.cpp]:
```suggestion
    ROS_INFO("Preparing joint %s for actuation", this->name.c_str());
```

**Review by Ishadijcks — COMMENTED** [review]
With a pr like this, it is usually a good idea to include a before and after of the logs. This makes it much easier to review what has actually changed.

I think you added the indentation to make it clearer that prints 'belong' to a specific joint, but it's hard to tell without an example.

**Comment by TimBuckers** [comment]
image
looks good

**Review by TimBuckers — APPROVED** [review]

**Comment by Ishadijcks** [comment]
@martijnvandermarel That looks very good indeed :)

**Pull request merged**